src/local/butler/deploy.py

```diff
@@ -65,6 +65,23 @@ def _get_services(paths):
   return services
 
 
+def _get_redis_ip(project):
+  """Get the redis IP address."""
+  region = appengine.region(project)
+  _, ip = common.execute('gcloud redis instances describe redis-instance '
+                         '--project={project} --region={region} '
+                         '--format="value(host)"'.format(
+                             project=project, region=region))
+  return ip.strip()
```

Comment:
check return code of common.execute, if failed, return None ?

Reply:
Raised an exception here. We can't continue in that case.